jsk_perception/cfg/ConcaveHullMaskImage.cfg

```diff
@@ -0,0 +1,12 @@
+#! /usr/bin/env python
+
+PACKAGE='jsk_perception'
+
+from dynamic_reconfigure.parameter_generator_catkin import *
+
+gen = ParameterGenerator()
+
+gen.add("min_area", double_t, 0, "minimum area of mask region", 0, 0.0, 10000000)
+gen.add("max_area", double_t, 0, "maximum area of mask region", 10000000, 0.0, 10000000)
```

Comment:
1. set -1 as the default of max_size in cfg.
2. double max_size = max_size_; if (max_size_ < 0) { max_size = height * width; } in cpp